lms/djangoapps/ccx/utils.py

```diff
@@ -268,22 +275,25 @@ def ccx_students_enrolling_center(action, identifiers, email_students, course_ke
                 errors.append(u"{0}".format(exp))
                 continue
 
-            # Check the License limit in the pearson course operation plugin.
-            if (
-                    configuration_helpers.get_value('PCO_ENABLE_LICENSE_ENFORCEMENT', False)
-                    and is_ccx_course(course_key)
-                    and not must_enroll
-               ):
-                allow_enrollment, message = run_extension_point(
-                    'PCO_ENFORCE_LICENSE_LIMITS',
-                    course_key=course_key,
-                )
+            if is_course_licensing_enable and not must_enroll:
+                # Check the License limit in the pearson course operation plugin.
+                allow_enrollment, message = run_extension_point('PCO_ENFORCE_LICENSE_LIMITS', course_key=course_key)
 
                 if not allow_enrollment:
                     log.info(message)
                     errors.append(message)
                     break
 
+                enrollment_counter += 1
+                messages.add_message(
```

Comment:
What do you think about moving the logic to calculate the error message to course_operations?